### DISCUSSION TITLE
How to start an open source project as a new member in Github?

### DISCUSSION THREAD

**Discussion by Isuru-Eranda** [community/community#72204] [New to GitHub]
Hi 👋, I'm Isuru Eranda

**Comment by Chanodyapraveen** [comment]
You should find the open source project in github. After that, you should frock them and solve their issues

**Reply by Isuru-Eranda** [reply]
Thank you very much for your reply

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
